lumigator/python/mzai/backend/backend/tests/integration/api/routes/test_api_workflows.py

```diff
@@ -30,38 +38,101 @@ def test_upload_data_launch_job(local_client: TestClient, dialog_dataset):
     assert create_response.status_code == 201
 
     created_dataset = DatasetResponse.model_validate(create_response.json())
+
     headers = {
         "accept": "application/json",
         "Content-Type": "application/json",
     }
-    payload = {
+    eval_payload = {
         "name": "test_run_hugging_face",
         "description": "Test run for Huggingface model",
-        "model": "hf://facebook/bart-large-cnn",
+        "model": "hf://hf-internal-testing/tiny-random-LlamaForCausalLM",
```

Comment:
Random model to be used in integration test. Does not return useful inferences, but otherwise behaves as a valid model.